feedback_prompt_learning/data/task/bigbench.py

```
@@ -0,0 +1,204 @@
+    
+    def _transform_epistemic(self, data):
+        original_examples = data['examples']
+        examples = []
+        
+        for example in original_examples:
+            question = example['input']
+            target_scores = example['target_scores']
+            
+            # Generating options and answer
+            options = list(target_scores.keys())
+            answer = [option.lower() for i, option in enumerate(options) if target_scores[option] == 1][0]
```

Ruff (0.14.7)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)